golang/internal/dockerhelper/container.go

```diff
@@ -0,0 +1,109 @@
+package dockerhelper
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/dyrector-io/dyrectorio/golang/internal/dogger"
+)
+
+const dockerClientTimeoutSeconds = 30
+
+func DeleteContainerByName(ctx context.Context, dog *dogger.DeploymentLogger, nameFilter string) error {
```

Comment:
No usage of  `dog *dogger.DeploymentLogger` in the function.

Reply:
dogger wasnt implemented in this draft yet.